README.md

```diff
@@ -164,7 +153,45 @@ For more parameters, please see [Monasca Agent Documentation](https://github.com
 
 This will create and run a new service file `/etc/systemd/system/monasca-agent.service` with the configuration set as per the arguments mentioned above. 
 
-### Log agent
+### Uninstalling Metrics agent
+Stop the services: 
+```
+systemctl stop monasca-agent
+```
+Delete created files: 
```

Comment:
Other files created by installer (and should be removed I think)
- /opt/monasca-agent
- /etc/sudoers.d/mon-agent

Reply:
Done